src/app/content/highlights/components/SummaryPopup/Filters.tsx

```diff
@@ -71,27 +108,28 @@ const Filters = ({className}: Props) => <div className={className}>
 </div>;
 
 export default styled(Filters)`
+  position: relative;
+  z-index: 2;
   overflow: visible;
   display: flex;
   flex-flow: row wrap;
   align-items: center;
-  padding: 2rem ${popupPadding}rem 0 ${popupPadding}rem;
   background: ${theme.color.neutral.base};
   border-bottom: 1px solid ${theme.color.neutral.formBorder};
-  ${theme.breakpoints.mobile(css`
-    padding: ${mobilePaddingTopBottom}rem ${mobilePaddingSides}rem;
-  `)}
-
   ${css`
     ${DropdownToggle} {
       font-weight: bold;
     }
 
-    ${Dropdown}:first-of-type {
-      margin-right: 8rem;
-      ${theme.breakpoints.mobile(css`
-        margin-right: 4.8rem;
-      `)}
+    ${Dropdown} {
+      & > *:not(${DropdownToggle}) {
+        top: calc(100% - 0.1rem);
+        box-shadow: 0 0 0.6rem 0 rgba(0, 0, 0, 0.2);
+        max-height: calc(100vh - 26rem);
+        ${theme.breakpoints.mobile(css`
+          max-height: calc(100vh - 19.6rem);
```

Comment:
i assume `0.1rem` is the bottom border width, `26rem` is the top and bottom space around the modal for desktop and `19.6rem` is the top and bottom space for mobile, but all of those magic numbers need to be extracted or derived from the other style constants

Reply:
It felt to me that hardcoding them would be easier to maintain, but this might not be the case. I've extracted the most required to the constants.

Reply:
if they're one off numbers i think its fine to hardcode them. but i felt like these were derivations on the modal padding and should probably be generated by some math involving those numbers, that way if we change the modal positioning we won't have to update the filters too